setup.py

```diff
@@ -66,6 +66,7 @@
         'Programming Language :: Python :: 3.6',
```

Comment:
Any reason not to drop 3.5 at the same time?

Reply:
Nope, 4e6eace6724c5d09e7cfa50e2144fc4344e16ce6